### PR TITLE
migrate to spring boot

### PR DESCRIPTION
**Pull request by randrii**

### CODE REVIEW

File: src/main/java/com/rybka/configuration/ApplicationConfiguration.java [hunk 1]
```diff
@@ -17,8 +18,7 @@
 
 @Configuration
```
Comment (jarc0der) [inline src/main/java/com/rybka/configuration/ApplicationConfiguration.java:18]: Do you need this configurations? 
SpingBootApplication provides all of them. 
Take a look into the annotation/

Comment (randrii) [reply src/main/java/com/rybka/configuration/ApplicationConfiguration.java:18]: fixed

File: src/test/java/com/rybka/service/connector/ExchangeRateConnectorTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest
```
Comment (jarc0der) [inline src/test/java/com/rybka/service/connector/ExchangeRateConnectorTest.java]: Use ExtendWith anotation

File: src/test/java/com/rybka/service/connector/ExchangeRateConnectorTest.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+class ExchangeRateConnectorTest {
```
Comment (jarc0der) [inline src/test/java/com/rybka/service/connector/ExchangeRateConnectorTest.java]: Rename to ExchangeServiceTest

File: src/test/java/com/rybka/service/connector/ExchangeRateConnectorTest.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+    @Mock
+    ExchangeRateConnector exchangeRateConnector;
+    @Mock
+    static ExchangeProperty exchangeProperty;
```
Comment (jarc0der) [inline src/test/java/com/rybka/service/connector/ExchangeRateConnectorTest.java]: make it non static.
and setUp also non static.

File: src/test/java/com/rybka/service/connector/PrimeExchangeRateConnectorTest.java [hunk 5]
```diff
@@ -0,0 +1,62 @@
+package com.rybka.service.connector;
```
Comment (jarc0der) [inline src/test/java/com/rybka/service/connector/PrimeExchangeRateConnectorTest.java]: Delete this test

Comment (jarc0der) [reply src/test/java/com/rybka/service/connector/PrimeExchangeRateConnectorTest.java]: Add test cases for calculateTotal and retrieveTopCurrency

**Pull request merged**